packages/components/src/theme/ComposerTheme/Typography/typography.ts

```diff
@@ -0,0 +1,17 @@
+export const defaultTypography = {
```

Comment:
where are this values coming from?, I cannot find them in figma

Reply:
as you can see [here](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/foundations/typography.ts) some default font sizes defined on ChakraUI are not totally aligned with some text styles of our theme. In the case of `font-body-meta-regular` defined line-height of `0.88rem` that we usually use on small components and in the default small fontSize  is defined as `0.875rem`.  The reason to do this is that when a Chakra component defined its own values those values have to be used or overwrite. The chakra input component [Input](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/input.ts) defined the fontSize attribute per size, in this case, to be aligned with the values defined on figma we can overwrite fontSize directly in our Input component because textStyle attribute is not going to replace it or set the global values as I did.